docs/section23/Rule23-2.md

```diff
@@ -0,0 +1,46 @@
+
+# Airside System - Rule 23-2  
+
+**Schema Version:** 0.0.12  
+**Mandatory Rule:** True  
+**Rule ID:** 23-2  
+**Rule Description:** System 5 & 7 minimum volume setpoint shall be 30% of zone peak airflow, minimum outdoor airflow, or rate required to comply with minium accreditation standards whichever is larger.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 23 Air-side  
+**90.1 Section Reference:** Section G3.1.3.13 VAV Minimum Flow Set Points (Systems 5 and 7)  
+**Data Lookup:** None  
+**Evaluation Context:** Building  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-5, 7, 7a, 5b, 7b, 7c.
+
+**Function Calls:**  
+
+1. get_baseline_system_types()
+2. is_baseline_system_type()
```

Comment:
I think get_baseline_system_types() should return baseline HVAC system dictionary in the format like (System 1{HVACID1, HVACID2...), System 2 (HVACID2, HVACID3, ...)} etc. Then we won't need to additionally call is_baseline_system_type

Reply:
need to discuss this comment, not sure I understand it